plugin-server/src/utils/db/postgres-logs-wrapper.ts

```diff
@@ -36,8 +36,9 @@ export class PostgresLogsWrapper {
             this.flushTimeout = null
         }
         if (this.logs.length > 0) {
-            await this.db.batchInsertPostgresLogs(this.logs)
+            const logs = this.logs
             this.logs = []
+            await this.db.batchInsertPostgresLogs(logs)
```

Comment:
Flyby fix to reduce a lot of noise (and errors?) in test logs when this flush gets triggered too quickly.